### ISSUE TITLE
fast5 files missing event data

### ISSUE THREAD

**Issue by jholzwa** [jts/nanopolish#737]
I am trying to use nanopolish, but my fast5 files don't include event data. I tried to basecall again using guppy with the --fast5_out switch, to no avail. Can I run nanopolish without event data? What is the option to do this (if available)? Or is there a tool to create this?

**Comment by jts** [comment]
You don't need events in the fast5 files to use nanopolish. Please see here for a discussion: http://simpsonlab.github.io/2017/09/06/nanopolish-v0.8.0/

**Comment by jholzwa** [comment]
Ok, then my fast5 files must still be a bit strange (as were my fastq files). I thought it was due to missing event data. 
The first time I tried to use nanopolish index, I got an error about there not being any fast5 files. I determined that my fastq files did not have the fast5 file name in the comment, so I wrote a small script to add this. 

When I attempted to run nanopolish to call the variants, I got the following error (one for each file):
[warning] fast5 file is unreadable and will be skipped: FAL49473_8d16791a33dea3e039270709b7df816bde0345e4_129.fast5

I had a look at the structure of the fast5 and concluded that I was missing the event data. So there must be some other problem with my files. We are using the minion with a minIT doing the base calling. Perhaps there is some option missing for the output files.

**Comment by jts** [comment]
>  I determined that my fastq files did not have the fast5 file name in the comment, so I wrote a small script to add this.

The fast5 name does not need to be a comment in the fastq file. Can you show me the full `nanopolish index` command that you tried to run? Which version of nanopolish do you have (`nanopolish --version`)

**Comment by jholzwa** [comment]
nanopolish index -d 20190923_1625_MN32191_FAL49473_f20bd036/fastq_pass barcode01.fastq

nanopolish version 0.10.2
Written by Jared Simpson.

Copyright 2015-2017 Ontario Institute for Cancer Research

I trimmed some of the paths (they were absolute and pretty long), because I don't think that makes any difference. I can put the entire path if I you do need them.

  I had used guppy to demultiplex the files, and this was the output file from guppy (barcode01.fastq).
